cli/popper/commands/cmd_dot.py

```diff
@@ -0,0 +1,74 @@
+import os
+
+import click
+import hcl
+import popper.utils as pu
+from ..cli import pass_context
+import types
+
+
+@click.option(
+    '--wfile',
+    help=(
+        'File containing the definition of the workflow. '
+        '[default: ./github/main.workflow OR ./main.workflow]'
+    ),
+    required=False,
+    default=None
+)
+@click.option(
+    '--recursive',
+    help='Run any .workflow file found recursively from current path. ',
```

Comment:
this description also needs to be updated since `popper dot` is not actually running the workflows

Reply:
Thanks! I missed this